src/index.js

```diff
@@ -1,32 +1,40 @@
-var XMLHttpRequest = require("xmlhttprequest").XMLHttpRequest;
+//Usé el browser para correr y corregir este script. Razón por la cual el require no está presente.
 
 var API = 'https://rickandmortyapi.com/api/character/';
```

Comment:
En el problema 2 debíamos utilizar ECMAScript 6 por lo tanto ya no utilizamos 'var'

```suggestion
const API = `https://rickandmortyapi.com/api/character/`;
```